website/src/releases/2023-02/deployment-previews.mdx

```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-03-03
+title: Custom Applications deployment previews
+description: Introducing deployment previews for Custom Applications.
+type: announcement
+topics:
+  - Merchant Center
+  - Configuration
+---
+
+Custom Applications now let you generate [Deployment previews](../concepts/deployment-previews), where you can test new features before publishing your application to the production environment.
+
+Emulating Custom Application previews with the [legacy workaround](../help-needed#legacy-workaround) required you to create multiple Custom Applications and dynamic configurations using environment variables. Deployment previews work without creating a new Custom Application in the Merchant Center, leading to less maintenance and better developer experience.
```

Comment:
```suggestion
Emulating Custom Application previews with the [legacy workaround](/help-needed#legacy-workaround) required you to create multiple Custom Applications and dynamic configurations using environment variables. Deployment previews work without creating a new Custom Application in the Merchant Center, leading to less maintenance and better developer experience.
```

Reply:
Nit: the "dynamic configurations using environment variables" is still needed for the URL part, I would maybe rephrase this a bit.

```suggestion
If you were using the [legacy workaround](../help-needed#legacy-workaround) by creating multiple Custom Applications with different entry points, we recommend migrating  to deployment previews where you only need to configure one Custom Application. This aims to provide less maintenance and a better developer experience.
```